shiny/api-examples/theme/requirements.txt

```diff
@@ -0,0 +1,2 @@
+shiny @ git+https://github.com/posit-dev/py-shiny.git@feat/dynamic-theming
```

Comment:
Remember to change this post merge

Reply:
Thanks, probably can just remove this file